infogami/utils/delegate.py

```diff
@@ -65,7 +65,7 @@ def layout_processor(handler):
     if out is None:
         out = RawText("")
 
-    if isinstance(out, basestring):
+    if isinstance(out, str):
```

Comment:
2to3 is dangerous because if breaks Python 2 compatibility to achieve Python 3 compatibility.

On Python 2 __isintance(u"Unicode", basestring)__ is True but __isintance(u"Unicode", str)__ is False.